datafusion/physical-expr/src/lib.rs

```diff
@@ -28,7 +28,6 @@ pub mod equivalence;
 pub mod execution_props;
 pub mod expressions;
 pub mod functions;
-pub mod hash_utils;
```

Comment:
I think this is technically a breaking change if someone used `hash_utils` directly in their code.

Perhaps we could  leave a pointer like

```suggestion
// backwards compatibility
pub use datafusion_common::hash_util;
```

But I don't think that is critical